lib/src/editor/ui/doc_markdown.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+import 'package:markdown/markdown.dart' as md;
+
+class DocMarkDown extends StatefulWidget {
+  final String? docPath;
+
+  const DocMarkDown({
+    Key? key,
+    this.docPath,
+  }) : super(key: key);
+
+  @override
+  _DocMarkDownState createState() => _DocMarkDownState();
+}
+
+class _DocMarkDownState extends State<DocMarkDown> {
+  @override
+  Widget build(BuildContext context) {
+    final controller = ScrollController();
+    const String _markdownData = """
+## Could Not Find Doc For Documents
```

Comment:
⛏️ - Maybe think about using `part` and `part of` and have this default text in separate file? Would simplify here, but if this would expand in the future it would already be in its own file.